depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,21 @@
+package=rapidcheck
+
+$(package)_version:1.0
+
+$(package)_download_path:https://github.com/Christewart/rapidcheck/releases/download/1.0
```

Comment:
Not sure what you mean about the deterministic bump. This is just a source tarball. Verifying would be no different than any of the other depends.

The general rule of thumb, though, is whether builders will be able to use it without any extra work. Since rapidcheck isn't present in any distros (as far as I can see), the answer to that is "no".

So, unless we're willing to require a depends build to run the new tests, a subtree would be easiest. However.

Tests are a bit of a special case. It may be reasonable to keep this in depends for now and let the c-i run it until there's sufficient coverage.

There's also the fact that it uses CMake, which we really don't want to introduce into our build (nothing against CMake, we just don't want to require every build tool under the sun).